src/content/ui-api/kubectl-gs/update-cluster.md

```diff
@@ -0,0 +1,47 @@
+---
+linkTitle: update cluster 
+title: "'kubectl gs update cluster' command reference"
+description: Reference documentation on how to update a Cluster using 'kubectl gs'.
+weight: 100
+menu:
+  main:
+    parent: uiapi-kubectlgs
+aliases:
+  - /reference/kubectl-gs/update-cluster/
+last_review_date: 2021-12-13
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-phoenix
+user_questions:
+  - How can I update a cluster from the command line?
+---
+
+# `kubectl gs update cluster`
+
+Updating clusters can be done in two ways. A cluster can be immediately updated or a cluster update can be scheduled in the future.
+
+## Usage
+
+### Update a cluster immediately
+
+```nohighlight
+kgs update cluster --name a1b2c --namespace org --release-version 16.1.0 --provider aws
```

Comment:
- We don't use the `kgs` alias any more.
- Long commands should be broken into one flag per line
- Better use a more realistic example namespace

```suggestion
kubectl gs update cluster \
  --name a1b2c \
  --namespace org-acme \
  --release-version 16.1.0 \
  --provider aws
```